qemu/tests/slof_hugepage.py

```diff
@@ -40,6 +40,7 @@ def _setup_hugepage(params):
     error_context.context('mount hugepages to %s'
                           % huge_page.hugepage_path, logging.info)
     huge_page.mount_hugepage_fs()
+    params["hugepage_path"] = huge_page.hugepage_path
```

Comment:
Did you test this one, for ppc platform ?

Reply:
Yes, passed with qemu4.2 and 6.0.